viadot/tasks/hubspot.py

```diff
@@ -0,0 +1,160 @@
+import json
+import pandas as pd
+import datetime
+
+from typing import List, Dict, Any, Literal
+from prefect import Task
+from prefect.tasks.secrets import PrefectSecret
+from prefect.utilities import logging
+from viadot.exceptions import ValidationError
+from viadot.sources import Hubspot
+
+logger = logging.get_logger()
+
+
+class HubspotToDF(Task):
+    def __init__(
+        self,
+        hubspot_credentials: dict,
+        *args,
+        **kwargs,
+    ):
+
+        self.credentials = hubspot_credentials
+        super().__init__(
+            name="hubspot_to_df",
+            *args,
+            **kwargs,
+        )
+
+    def __call__(self):
+        """Download Hubspot data to a DF"""
+        super().__call__(self)
+
+    def to_df(self, result: list = None) -> pd.DataFrame:
+        """
+        Args:
+            result (list, optional): API response in JSON format. Defaults to None.
+
+        Returns:
+            pd.DataFrame: DF from JSON
+        """
+        return pd.json_normalize(result)
+
+    def date_to_unixtimestamp(self, date: str = None):
```

Comment:
Add `date` to the docstring under `Args:` section.
Add the return type for this method and `Returns:` section.